neofetch

```diff
@@ -2721,6 +2722,22 @@ get_memory() {
     esac
 }
 
+get_bootloader() {
+    case $os in
+        "Linux")
+            if [[ -d "/boot/grub" ]]; then
+                bootloader="GRUB ($(grub-install --version | awk '{printf $3}'))"
```

Comment:
Same here, the command is `grub2-install`, and it needs root privileges on openSUSE.